Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -265,11 +268,7 @@ func failureBreakpoint() {
   // behavior can be disabled by passing the `-no_deduplicate` flag described in
   // ld(1), but that would disable it module-wide and sacrifice optimization
   // opportunities elsewhere. Instead, this function performs a trivial
-  // function call, passing it a sufficiently unique value to avoid
-  // de-duplication.
-  struct NoOp {
-    nonisolated(unsafe) static var ignored: Int = 0
-    static func perform(_: inout Int) {}
-  }
-  NoOp.perform(&NoOp.ignored)
+  // operation on a usable-from-inline value, which the compiler must assume
+  // cannot be optimized away.
+  failureBreakpointValue = 1
```

Comment:
If we were to enable the Thread Sanitizer and run our test suite, would it flag this as a data race? (Despite always assigning the same value)

Reply:
Nope, but if it did I would expect the original implementation would have the same issue.

Reply:
I'd be surprised if both of them did not report a TSan race. But you're saying you tried and they did not? 🤔

Either way, this PR seems fine. I just really want to avoid transforming this into a global incrementing counter in the future, since then it would require a lock and would increase the overhead of every test failure.